dlang.org.ddoc

```diff
@@ -301,9 +301,9 @@ WIKI=$(TITLE)
 _=
 
 XREF=$(XREF2 $1, $2, $(D std.$1.$2))
-XREF2=$(SPANC libref, $(AHTTP dlang.org/phobos/std_$1.html#$2, $(TAIL $+)))
+XREF2=$(SPANC libref, $(A /phobos/std_$1.html#$2, $(TAIL $+)))
 XREF_PACK=$(XREF_PACK_NAMED $1, $2, $3, $(D std.$1.$2.$3))
-XREF_PACK_NAMED=$(SPANC libref, $(AHTTP dlang.org/phobos/std_$1_$2.html#$3, $4))
+XREF_PACK_NAMED=$(SPANC libref, $(A /phobos/std_$1_$2.html#$3, $4))
```

Comment:
You're quite right, it messes up the path when viewed locally. It also messes up on a local webserver where it isn't installed to the web root.

Fixed. Thanks for the tip about `$(ROOT_DIR)`, I wasn't aware of it before.